ckanext/switzerland/tests/test_controller.py

```diff
@@ -38,7 +38,7 @@ def setup(self):
         # create a valid DCAT-AP Switzerland compliant dataset
         self.dataset = {
             'coverage': '',
-            'issued': '2015-09-08T00:00:00',
+            'issued': '08.09.2015',
```

Comment:
The dates must be in ISO-Format in the API and XML. So I am not quite sure whether that will still be the case with that change. Did you check this?

Reply:
The format in the API is handled by the output_validators, not the input validators, and the XML export is handled by ckanext-dcatapchharvest, so neither of them is affected by this change.